### PR TITLE
Add corrections for space(s).

### PR DESCRIPTION
**Pull request by cfi-gb**
See e.g. the following (not much hits but still happening from time to time):
- https://grep.app/search?q=speace&words=true
- https://grep.app/search?q=spaece&words=true
- https://grep.app/search?q=spaeces&words=true

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -29203,6 +29205,8 @@ spcifications->specifications
 spcified->specified
 spcifies->specifies
 spcify->specify
+speace->space
+speaces->spaces
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:29209]: Again please.

**Review by peternewman — CHANGES_REQUESTED** [review]
Some suggested improvements.

I was expecting a load of two word corrections at first. 😆

**Comment by cfi-gb** [comment]
> Some suggested improvements.

Thanks a lot for your review. I have now added the ed/es/ing variant for all newly added but also for the existing ones. Hope i got everything right.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -24831,7 +24831,9 @@ prtinf->printf
 prufe->proof
 prviate->private
 psace->space
+psaced->spaced
 psaces->spaces
+psacing->spacing
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:24836]: An alternative option:
```suggestion
psace->space, pace,
psaced->spaced, paced,
psaces->spaces, paces,
psacing->spacing, pacing,
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -29231,8 +29240,12 @@ spcifications->specifications
 spcified->specified
 spcifies->specifies
 spcify->specify
+speace->space
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:29243]:
```suggestion
speace->space, peace,
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -29231,8 +29240,12 @@ spcifications->specifications
...
+speaced->spaced
+speaces->spaces
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:29245]: Maybe (phonetically):
```suggestion
speaces->spaces, species,
```

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -29677,6 +29690,9 @@ splitted->split
 splittng->splitting
 spllitting->splitting
 spoace->space
+spoaced->spaced
+spoaces->spaces
+spoacing->spacing
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:29695]: I did wonder about poach... but probably unlikely.

Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt:29695]: Yes, indeed this looks to me quite unlikely so hope it is o.K. if i'm resolving this conversation now.

**Pull request merged**